backend/repositories/match-repository.js

```diff
@@ -22,8 +22,32 @@ const storeMatch = async (match) => {
     try {
         result = await storageContext.insertMatch(match)
 
-        await updateRatingForTeam(winningTeam, match.ratingChange, storageContext)
-        await updateRatingForTeam(losingTeam, -match.ratingChange, storageContext)
+        // FIXME: foosbot's report of rating change is incorrect if the number of players on each side is not the same
+        // one player beat two players
```

Comment:
The condition says `winningTeam.length < losingTeam.length` which doesn't necessarily mean that `one player beat two players`. I would either make the condition more specific or change the calculation to take into the account different numbers of players in the team to save us potential trouble down the line.

Reply:
Based on the documentation it does. Take a look at this:
```
  * @param {Array<number>} matchDescription.team1 Array of 1 or 2 elements containing IDs of players from team1.
  * @param {Array<number>} matchDescription.team2 Array of 1 or 2 elements containing IDs of players from team2.
```
https://github.com/salsita/foosball-rating/blame/master/backend/repositories/match-repository.js#L89
But sure, you are right. I knew about it as I was implementing it but I really didn't expect everyone to suddenly start taking it so seriously... As I said above, not sure when I'll have the time (and the will) to change all this...